cranelift/2023/cranelift-12-27.md

```diff
@@ -0,0 +1,21 @@
+# December 27 project call
```

Comment:
We can probably preemptively cancel/delete this one -- I doubt most folks will be around during the holiday week?

Reply:
Good idea!